classes/requests/post/class-truelayer-request-create-payment.php

```diff
@@ -24,143 +33,171 @@ class TrueLayer_Request_Create_Payment extends TrueLayer_Request_Post {
 	 */
 	public function __construct( $arguments ) {
 		parent::__construct( $arguments );
-		$this->log_title       = 'Create payment';
-		$this->endpoint        = '/payments';
-		$this->idempotency_key = Truelayer_Helper_Signing::get_uuid();
-		$this->order_id        = $arguments['order_id'];
+		$this->log_title = 'Create payment';
+		$this->order_id  = $arguments['order_id'];
```

Comment:
As mentioned for the review of `process_payment`. We're passing a `WC_Order` but assume an order ID here.